docs/getting-started/installation.md

```diff
@@ -76,44 +116,10 @@ gremlin> :remote connect tinkerpop.server conf/remote.yaml
 As you can probably tell from the log, the client and server are running on the same machine in this case.
 When using a different setup, all you have to do is modify the parameters in the `conf/remote.yaml` file.
 
-Running JanusGraph inside a Docker container
---------------------------------------------
-
-For virtualization, JanusGraph also provides a [docker image](https://hub.docker.com/r/janusgraph/janusgraph).
-Docker makes it easier to elaborate and evaluate differend configurations of servers and clients on a single machine.
-Let's try running a simple JanusGraph instance in Docker:
-```bash
-$ docker run -it -p 8182:8182 janusgraph/janusgraph:latest
-```
-We run the image interactively and tell docker to make it's port 8182 available for us to see.
-The server may need a few seconds to start up so be patient and wait for the corresponding log messages to appear.
-We can now start a Gremlin console on our local device and try to connect to the new server:
-```bash
-$ bin/gremlin.sh
+!!! warning
+    The above command only establishes the connection to the server.
+    It does not forward the following commands to the server by default!
+    As a result, further commands will still be executed locally unless preceeded by `:>`.
 
-         \,,,/
-         (o o)
------oOOo-(3)-oOOo-----
-gremlin> :remote connect tinkerpop.server conf/remote.yaml
-==>Configured localhost/127.0.0.1:8182
-```
-Notice that the client side of this works exactly the same as before when running both the client and server locally without docker.
-
-Conveniently, it's also possible to run both the server and the client within separate Docker containers.
-We therefore instantiate a container for the server:
-```bash
-$ docker run --name janusgraph-default janusgraph/janusgraph:latest
-```
-We can now instruct Docker to start a second container for the client and link it to the already running server.
-```bash
-$ docker run --rm --link janusgraph-default:janusgraph -e GREMLIN_REMOTE_HOSTS=janusgraph \
-    -it janusgraph/janusgraph:latest ./bin/gremlin.sh
-
-         \,,,/
-         (o o)
------oOOo-(3)-oOOo-----
-gremlin> :remote connect tinkerpop.server conf/remote.yaml
-==>Configured janusgraph/172.17.0.2:8182
-```
-Notice how it's not necessary to bind any ports in order to make this example work.
-For further reading, see the [JanusGraph Server](../basics/server.md) section.
+    To forward every command to the remote server, use the `:remote console` command.
+    Further documentation can be found in the (TinkerPop reference docs)[https://tinkerpop.apache.org/docs/current/reference/#console-remote-console]
```

Comment:
Please don't link to `current`, but use `{{ tinkerpop_version }}` instead. We should always link to the docs corresponding to the version we support in the respective JanusGraph version.